src/main/java/seedu/address/logic/commands/edit/EditInternshipCommand.java

```diff
@@ -89,7 +91,7 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         return new EditCommandResult(editedInternship.toString(),
-                String.format(MESSAGE_EDIT_INTERNSHIP_SUCCESS, editedInternship),
+                String.format(MESSAGE_EDIT_INTERNSHIP_SUCCESS, editedInternship.getType().getFullType()),
```

Comment:
This means that the message will be "Edited Internship: Internship" Not sure if you want to do that? Maybe editedInternship.getName().fullname?

Reply:
Got it!